tests/unit/test_circular_imports.py

```diff
@@ -43,7 +44,12 @@ def _discover_path_importables(
     pkg_pth: Path,
     pkg_name: str,
 ) -> Generator[str, None, None]:
-    """Yield all importables under a given path and package."""
+    """Yield all importables under a given path and package.
+
+    :param pkg_pth: The path to the package to walk
+    :param pkg_name: The name of the package
+    :yields: A generator of package directory paths and prefixes
```

Comment:
No, this function itself is a generator but it doesn't yield other generators, it yields paths from a generator expression.
```suggestion
    :yields: Package directory paths
```
Also, the prefix attachment is an implementation detail that is not significant here.